modules/insticatorBidAdapter.js

```diff
@@ -170,6 +170,18 @@ function buildImpression(bidRequest) {
     },
   }
 
+  let bidFloor = parseFloat(deepAccess(bidRequest, 'params.floor'));
+
+  if (!isNaN(bidFloor)) {
+    imp.bidfloor = deepAccess(bidRequest, 'params.floor');
+    imp.bidfloorcur = 'USD';
+    const bidfloorcur = deepAccess(bidRequest, 'params.bidfloorcur')
+    if (bidfloorcur && bidfloorcur !== 'USD') {
+      delete imp.bidfloor;
+      delete imp.bidfloorcur;
+    }
```

Comment:
Please include the new params `bidfloor` and `bidfloorcur` to your markdown file. A mention that only `USD` is accepted would be useful, along with a logWarn here

Reply:
Updated the description in Docs https://github.com/prebid/prebid.github.io/pull/5307 PR.